packages/app/src/providers/mock/mock-api.ts

```diff
@@ -67,7 +67,7 @@ export class MockApi extends DataProviderApi {
       throw new Error('error');
     }
 
-    if (dataset.name.startsWith('slow')) {
+    if (dataset.name.startsWith('slow') && abortSignal) {
```

Comment:
I've made the `abortSignal` required in `cancellableDelay` to simplify the code. In practice, we never request the `'slow_value'` dataset without an abort signal (it's optional to simplify testing the h5grove and h5wasm APIs).